cmd/erasure-server-pool-decom.go

```diff
@@ -570,15 +580,16 @@ func (v versionsSorter) reverse() {
 }
 
 func (z *erasureServerPools) decommissionPool(ctx context.Context, idx int, pool *erasureSets, bName string) error {
-	var forwardTo string
-	// If we resume to the same bucket, forward to last known item.
-	rbucket, robject := z.poolMeta.ResumeBucketObject(idx)
-	if rbucket != "" && rbucket == bName {
-		forwardTo = robject
-	}
+	var wg sync.WaitGroup
+	wStr := env.Get("_MINIO_DECOMMISSION_WORKERS", "12")
```

Comment:
is their number of sets 14?

Reply:
it's 12 @kannappanr

Reply:
Just use the len(pool.sets)

Reply:
This being a configurable value doesn't make sense even as a hotfix.

Reply:
having a configurable value will allow reducing concurrency if needed

Reply:
Then it should default to len(pool.sets) and then allow reducing. Not use 12 as hardcoded value. 

If this is also something we do not merge to master why send this as a PR, just build locally and give it to customer.

Reply:
we need to improve upon this a bit more before we can merge it to master. We can default to `len(pool.sets)`